code/modules/mob/living/living_say.dm

```diff
@@ -338,7 +341,9 @@ GLOBAL_LIST_INIT(message_modes_stat_limits, list(
 	var/whisper_range = 0
 	var/is_speaker_whispering = FALSE
 	if(message_mods[WHISPER_MODE]) //If we're whispering
-		whisper_range = EAVESDROP_EXTRA_RANGE
+		whisper_range = 6 // Needed for good hearing trait. The actual filtering for whispers happens at the /mob/living/Hear proc
```

Comment:
Pure Inacusiate already lets you hear whispers from far away, are we able to make this work the same way inacusiate does, or is this extra checks and more overhead requried?

Reply:
I just tested pure inacusiate and it doesn't seem to do anything at all. And it was on a clean branch of the code so it wasn't broken by my changes. Originally, I wanted to use signals like Inacusiate did, but it simply didn't work because the character replacement thing happens _before_ the relevant signal is called.

Reply:
To clarify, its not possible to make the ears work like pure inacusiate because pure inacusiate doesn't work. I could make pure inacusiate work the way ears do, but it would probably make more sense to do that in a separate PR.

Reply:
lol

Reply:
should make a new define or preferably update EAVESDROP_EXTRA_RANGE if imaginary_friend/send_speech() should be affected the same way